src/test/java/LoginTests.java

```diff
@@ -1,61 +1,42 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.testng.Assert;
-import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
-import pages.HomePage;
-import pages.LoginPage;
 
-import java.util.List;
-
-
-public class LoginTests extends BaseTest {
-
-
-    @Test(dataProvider = "IncorrectLoginProviders")
-    public void negativeLoginTests(String email, String password) {
-        LoginPage loginPage = new LoginPage(getThreadLocal());
-        loginPage.enterEmail(email);
-        loginPage.enterPassword(password);
-        loginPage.clickLoginBtn();
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
+import java.time.Duration;
 
+public class LoginTests  {
     @Test
-    public void loginSucceedTest() throws InterruptedException {
-        LoginPage loginPage = new LoginPage(getThreadLocal());
-        HomePage homePage = new HomePage(getThreadLocal());
-        loginPage.enterEmail("demo@class.com");
-        loginPage.enterPassword("te$t$tudent");
-        loginPage.clickLoginBtn();
-        Thread.sleep(3000);
-        Assert.assertTrue(homePage.getAvatar());
-    }
-
+    public static void loginSucceedTest()throws  InterruptedException {
+
+//      Added ChromeOptions argument below to fix websocket error
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        options.addArguments("--disable-notification*");
+
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        String url = "https://qa.koel.app/";
+        driver.get(url);
```

Comment:
Remove the line of code. Already defined in BaseTest. After extending the BaseTest, LoginTest will inherit the chrome options defined in BaseTest.